src/primitives/Btn.js

```diff
@@ -0,0 +1,39 @@
+/** @jsxImportSource @emotion/react */
+
+import { SHAPE } from '../tokens';
```

Comment:
Now what if, and hear me out here, we keep all tokens in CSS vars? That way there are no re-renders in react when tokens change and so far I have yet to come up to an issue with that thinking?